tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -198,12 +199,26 @@ Future<void> _main(List<String> args) async {
       var verbose = results.command!['verbose'];
       var watch = results.command!['watch'];
 
+      //TODO: set path for load from option
+      var config = GeneratorConfig.load();
+      if (config == null) {
+        return;
```

Comment:
No, since GeneratorConfig.load always prints before returning null.

Maybe it's not the best design, but I did not change it, because it worked like this before. The load method just returned a bool instead.

Make load always return a GeneratorConfig or throw something?